admin/routes/check-started.js

```diff
@@ -1,10 +1,19 @@
 const express = require('express')
 const router = express.Router()
+const moment = require('moment')
 const { checkStarted } = require('../controllers/check-started')
+const winstonLogger = require('./../winstonLogger')
 
 router.route('/').all((req, res) => {
-  if (req.method !== 'POST') return res.sendStatus(405)
+  const apiLogger = winstonLogger.apiLogger()
+  apiLogger.info(`MTC-API: check-started (${moment()}) reached`)
```

Comment:
you don't need to append the time, this is taken care of by LogDNA

Reply:
I added the time because I noticed a little bit of lag and thought of checking if it would display. I will remove it after I'm more clear about it.